### PR TITLE
XEP-0393: updates based on LC feedback

### PR DESCRIPTION
**Pull request by SamWhited**
Resubmitting with registrar updates, a disco feature, and the zero width space technique.

### CODE REVIEW

File: xep-0393.xml [hunk 1]
```diff
@@ -255,6 +267,19 @@
     </di>
   </dl>
 </section1>
+<section1 topic='Discovering Support' anchor='disco'>
+  <p>
+    Clients that support Extensible IBR MUST advertise the fact by including a
```
Comment (horazont) [inline xep-0393.xml:272]: Extensible IBR? I suppose this was meant to say "Message Styling"?

File: xep-0393.xml [hunk 2]
```diff
@@ -467,24 +525,67 @@
...
   </p>
   <p>
-    Styled text may also be rendered poorly by screen readers.
+    Styled text may be rendered poorly by screen readers.
     When applying formatting it may be desirable to include directives to
-    exclude styling directives from being read.
+    exclude styling directives from being read, or to add markers to the final
+    output that have semantic meaning for screen readers.
+    For example, in an web based client an emphasis span might be converted to
+    an HTML &lt;emph&gt; element.
```
Comment (horazont) [inline xep-0393.xml:533]: The HTML element which I think you’re thinking of is ``<em/>``, not ``<emph/>``.

File: xep-0393.xml (hunk 1 above)
Comment (SamWhited) [reply xep-0393.xml]: Oops, copy/pasta. Fixed. Thanks!

File: xep-0393.xml (hunk 2 above)
Comment (SamWhited) [reply xep-0393.xml]: Ah thanks, I don't do a lot of HTML but wanted an example people would be familiar with. Should have familiarized myself first. Fixed!

**Pull request merged**